automation_tools/__init__.py

```diff
@@ -382,6 +386,8 @@ def setup_firewall(definitions=None):
         if run('systemctl status firewalld', quiet=True).failed:
             run('systemctl enable firewalld')
             run('systemctl start firewalld')
+        if flush:
+            raise RuntimeWarning('flush for firewalld is not available')
```

Comment:
Tried to check how to do this but still have not found. Decided to send the PR this way to see if someone knows from the top of his/her head. Please do not merge before this line had changed to something that actually do the expected job.